docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc

```diff
@@ -131,6 +131,14 @@ When tasks are assigned to a group of users, you can use pluggable task assignme
 
 === Business rule task fire limit
 The business rule task has been enhanced to limit the number of fired rules. This avoids situations where rules run into an infinite loop and make the server completely unresponsive. The fire default fire limit is 10000.
```

Comment:
I would probably change "The fire default fire limit is 10000." to "The default fire limit is 10000."